src/quarkbaellchen.md

```diff
@@ -0,0 +1,32 @@
+# Quarkbällchen – Fried curd balls
+
+![An image of my finished curd balls](pix/quarkbaellchen.webp)
+
+“Quark” is a type of fresh dairy product. It's usually translated as curd cheese or cottage cheese.
+
+## Ingredients
+
+- 420ml quark (plain yoghurt can be used as a replacement)
+- 2 eggs
+- 6/7 spoons of sugar
```

Comment:
```suggestion
- 6 or 7 tbsp of sugar
```

Just to make it clear that it is a table spoon and not something smaller or bigger.